Core/src/IfcGeometryConverter/GeomUtils.h

```diff
@@ -62,93 +62,286 @@ namespace OpenInfraPlatform
+				*
+				* \return							Returns true, if lines are intersecting with each other. False otherwise.
+				*/
 				static bool LineSegmentToLineIntersection(const carve::geom::vector<2>& initialPointSeg1,
 					const carve::geom::vector<2>& terminalPointSeg1,
 					const carve::geom::vector<2>& initialPointSeg2,
 					const carve::geom::vector<2>& terminalPointSeg2,
 					carve::geom::vector<2> & intersectionPoint);
 
+				/**********************************************************************************************/
+				/*! \brief Calculates coordinates of the intersection point.
+				*
+				* \param[in] initialPointSeg1		Initial point of the first intersecting segment, which describes the direction of the line.
+				* \param[in] terminalPointSeg1		Terminal point of the first intersecting segment, which describes the direction of the line.
+				* \param[in] initialPointSeg2		Initial point of the second intersecting segment, which describes the direction of the line.
+				* \param[in] terminalPointSeg2		Terminal point of the second iintersecting segment, which describes the direction of the line.
+				* \param[out] intersectionPoint		Coordinates of the intersection point between first line and second line.
+				*
+				* \return							Returns true, if lines are intersecting with each other. False otherwise.
+				*/
 				static bool LineSegmentToLineSegmentIntersection(const carve::geom::vector<2>& initialPointSeg1,
 					const carve::geom::vector<2>& terminalPointSeg1,
 					const carve::geom::vector<2>& initialPointSeg2,
 					const carve::geom::vector<2>& terminalPointSeg2,
 					carve::geom::vector<2> & intersectionPoint);
 
+				/**********************************************************************************************/
+				/*! \brief Calculates coordinates of the intersection point.
+				*
+				* \param[in] initialPointSeg1		Initial point of the first intersecting line.
+				* \param[in] terminalPointSeg1		Terminal point of the first intersecting line.
+				* \param[in] initialPointSeg2		Initial point of the second intersecting line.
+				* \param[in] terminalPointSeg2		Terminal point of the second intersecting line.
+				* \param[out] intersectionPoint		Coordinates of the intersection point between first line segment and second line segment.
+				*
+				* \return							Returns true, if lines are intersecting with each other. False otherwise.
+				*/
 				static bool GeomUtils::LineToLineIntersection(const carve::geom::vector<2>& initialPointSeg1,
 					const carve::geom::vector<2>& terminalPointSeg1,
 					const carve::geom::vector<2>& initialPointSeg2,
 					const carve::geom::vector<2>& terminalPointSeg2,
 					carve::geom::vector<2> & intersectionPoint);
 
+				/**********************************************************************************************/
+				/*! \brief Appedns vector of points (in 3D) to the curve.

```

Comment:
```suggestion
				/*! \brief Appends vector of points (in 3D) to the curve.
```

Include a note, that duplication of points is checked and duplicate points are skipped.

Reply:
Resolved in: 
36ad1fc
51d6064